nextcord/message.py

```diff
@@ -781,6 +785,14 @@ def __init__(
             except KeyError:
                 continue
 
+        from .interactions import MessageInteraction
+
+        self.interaction: Optional[MessageInteraction]
+        try:
+            self.interaction = MessageInteraction(data=data["interaction"], state=state)
+        except KeyError:
+            self.interaction = None
+
```

Comment:
```suggestion
        self.interaction: Optional[MessageInteraction] = MessageInteraction(data.get("interaction")) if data.get("interaction") is not None else None

```

Reply:
Could you not use `if "interaction" in data` and ["item"]